src/drive/web/modules/actions/useActions.jsx

```diff
@@ -19,15 +19,21 @@ const withHideSelectionBarDispatch = (action, dispatch) => {
 const useActions = (actionCreators, actionOptions = {}) => {
   const dispatch = useDispatch()
   let actions = []
-  actionCreators.map(createAction => {
-    const enhancedAction = withHideSelectionBarDispatch(
-      createAction(actionOptions),
-      dispatch
-    )
-    const name = enhancedAction.label || enhancedAction.icon
-
-    actions.push({ [name]: enhancedAction })
-  })
+  actionCreators
+    .map(createAction => {
+      const action = createAction(actionOptions)
+
+      // Temporarily disable Download button on iOS Flagship app until
+      // the download feature is fixed on this platform
+      // When fixed on this platform, revert PR #2672
+      if (action === null) return null
+
+      const enhancedAction = withHideSelectionBarDispatch(action, dispatch)
+      const name = enhancedAction.label || enhancedAction.icon
+
+      actions.push({ [name]: enhancedAction })
+    })
+    .filter(x => x)
```

Comment:
i think this line is useless

Reply:
Good catch, I put it before realizing that it was a push inside of a map. I'll remove it